devicetypes/smartthings/nodon-lighting-switch.src/nodon-lighting-switch.groovy

```diff
@@ -0,0 +1,376 @@
+/*
+ *  Integrated ZigBee Switch
+ * 
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+public static String version() { return "v0.0.01.20220114" }
+
+import java.lang.Math
+import groovy.json.JsonOutput
+
+
+private getMODEL_MAP() { 
+    [
+
+        "SIN-4-2-20" : 2
+    ]
+}
+
+metadata {
+    definition(name: "NodOn lighting Switch", namespace: "Smartthings", author: "NodOn", ocfDeviceType: "oic.d.switch", vid: "generic-switch") {
```

Comment:
I'm assuming that everything works good with `zigbee-multi-switch`, so can you remove this file from the PR?

Reply:
@PKacprowiczS   From the beginning, I just forked the directory, and I've been working live ever since: https://graph-eu01-euwest1.api.smartthings.com/ide/devices. So I can't remove previous version from command line.

Reply:
I don't see what IDE has to do with removing a file from your branch/PR.

Reply:
@PKacprowiczS i delete it on my branch, thanks!